src/locales/ja/modifier-type.ts

```diff
@@ -0,0 +1,456 @@
+    },
+    "FusePokemonModifierType": {
+      description: "2匹のポケモンをけつごうする (とくせいをいどうし、きほんステータスとタイプをわけ、わざプールをきょうゆうする)",
+    },
+    "TerastallizeModifierType": {
+      name: "{{teraType}} Tera Shard",
+      description: "ポケモンを{{teraType}}タイプにテラスタル（10かいのバトルまで）",
+    },
+    "ContactHeldItemTransferChanceModifierType": {
+      description: "こうげきするとき あいてがもっているアイテムを {{chancePercent}}パーセントのかくりつでぬすむ",
+    },
+    "TurnHeldItemTransferModifierType": {
+      description: "まいターン あいてからひとつのもちものをてにいれる",
+    },
+    "EnemyAttackStatusEffectChanceModifierType": {
+      description: "こうげきわざに {{chancePercent}}パーセントのかくりつで {{statusEffect}}をあたえる",
+    },
+    "EnemyEndureChanceModifierType": {
+      description: "こうげきをこらえるかくりつを{{chancePercent}}パーセントふやす",
+    },
+
+    "RARE_CANDY": { name: "ふしぎなアメ" },
+    "RARER_CANDY": { name: "もっとふしぎなアメ" },
+
+    "MEGA_BRACELET": { name: "メガバングル", description: "メガストーンがつかえるようになる" },
+    "DYNAMAX_BAND": { name: "ダイマックスバンド", description: "ダイスープがつかえるようになる" },
+    "TERA_ORB": { name: "テラスタルオーブ", description: "テラピースがつかえるようになる" },
+
+    "MAP": { name: "ちず", description: "わかれみちでいきさきをえらべるようになる" },
+
+    "POTION": { name: "キズぐすり" },
+    "SUPER_POTION": { name: "いいキズぐすり" },
+    "HYPER_POTION": { name: "すごいキズぐすり" },
+    "MAX_POTION": { name: "まんたんのくすり" },
+    "FULL_RESTORE": { name: "かいふくのくすり" },
+
+    "REVIVE": { name: "げんきのかけら" },
+    "MAX_REVIVE": { name: "げんきのかたまり" },
+
+    "FULL_HEAL": { name: "なんでもなおし" },
+
+    "SACRED_ASH": { name: "せいなるはい" },
+
+    "REVIVER_SEED": { name: "ふっかつのタネ", description: "ひんしになったときもっているポケモンをHPはんぶんでふっかつさせる" },
+
+    "WHITE_HERB": { name: "White Herb", description: "An item to be held by a Pokémon. It will restore any lowered stat in battle." },
+
+    "ETHER": { name: "ピーピーエイド" },
+    "MAX_ETHER": { name: "ピーピーリカバー" },
+
+    "ELIXIR": { name: "ピーピーエイダー" },
+    "MAX_ELIXIR": { name: "ピーピーマックス" },
+
+    "PP_UP": { name: "ポイントアップ" },
+    "PP_MAX": { name: "ポイントマックス" },
+
+    "LURE": { name: "ダブルバトルコロン" },
+    "SUPER_LURE": { name: "シルバーコロン" },
+    "MAX_LURE": { name: "ゴールドコロン" },
+
+    "MEMORY_MUSHROOM": { name: "きおくキノコ", description: "ポケモンのわすれたわざをおぼえさせる" },
+
+    "EXP_SHARE": { name: "がくしゅうそうち", description: "バトルにさんかしていないポケモンが けいけんちの20パーセントをもらう" },
+    "EXP_BALANCE": { name: "バランスそうち", description: "レベルがひくいパーティメンバーがもらうけいけんちがふえる" },
+
+    "OVAL_CHARM": { name: "まるいおまもり", description: "バトルにふくすうのポケモンがさんかするとけいけんちが10パーセントふえる" },
+
+    "EXP_CHARM": { name: "けいけんちおまもり" },
+    "SUPER_EXP_CHARM": { name: "いいけいけんちおまもり" },
+    "GOLDEN_EXP_CHARM": { name: "ゴールドけいけんちおまもり" },
+
+    "LUCKY_EGG": { name: "しあわせタマゴ" },
+    "GOLDEN_EGG": { name: "おうごんタマゴ" },
+
+    "SOOTHE_BELL": { name: "やすらぎのすず" },
+
+    "SCOPE_LENS": { name: "Scope Lens", description: "It's a lens for scoping out weak points. It boosts the holder's critical-hit ratio."},
+    "LEEK": { name: "Leek", description: "This very long and stiff stalk of leek boosts the critical-hit ratio of Farfetch'd's moves."},
```

Comment:
```suggestion
    "SCOPE_LENS": { name: "ピントレンズ", description: "弱点が 見える レンズ。持たせた ポケモンの技が 急所に 当たりやすくなる。"},
    "LEEK": { name: "ながねぎ", description: "とても長くて 硬いクキ。カモネギに 持たせると 技が 急所に 当たりやすくなる。"},
```
Localization for `LEEK` and `SCOPE_LENS` using the same source from their initial PR (#2666).